python/operations.cpp

```diff
@@ -72,6 +73,23 @@ template <typename T> void bind_sort_dim(py::module &m) {
       doc.c_str());
 }
 
+template <typename T> void bind_sort_variable(py::module &m) {
+  m.def(
+      "sort",
+      [](const typename T::const_view_type &x, const Dim dim,
+         const std::string &order) {
+        if (order == "ascending")
+          return sort(x, dim, variable::SortOrder::Ascending);
+        else if (order == "descending")
+          return sort(x, dim, variable::SortOrder::Descending);
+        else
+          throw std::runtime_error(
+              "Sort order must be 'ascending' or 'descending'");
+      },
+      py::arg("x"), py::arg("dim"), py::arg("order") = "ascending",
+      py::call_guard<py::gil_scoped_release>());
```

Comment:
Could you add a docstring please? See examples in other places, there is a helper class called `Docstring`.